### ISSUE TITLE
XtraFinder.download Fails to Download AppCast URL

### ISSUE THREAD

**Issue by foigus** [autopkg/foigus-recipes#3]
XtraFinder.download fails:

```
$ autopkg run XtraFinder.download
Processing XtraFinder.download...
Can't open URL https://www.trankynam.com/xtrafinder/XtraFinder-Appcast.xml
Failed.

The following recipes failed:
    XtraFinder.download
        Error in com.github.foigus.download.XtraFinder: Processor: SparkleUpdateInfoProvider: Error: Can't open URL https://www.trankynam.com/xtrafinder/XtraFinder-Appcast.xml

Nothing downloaded, packaged or imported.
```

This appears to be related to [an issue](https://groups.google.com/forum/#!topic/autopkg-discuss/Yz8jKT4Taa0) with SparkleUpdateInfoProvider.  For the moment this could be addressed by overriding the SPARKLE_FEED_URL with:

http://www.trankynam.com/xtrafinder/XtraFinder-Appcast.xml

**Comment by foigus** [comment]
SparkleUpdateInfoProvder is being discussed/worked on [here](https://groups.google.com/forum/#!topic/autopkg-discuss/W32ACOOCQ9w)

**Comment by foigus** [comment]
Addressed by AutoPkg 0.6.0 and higher.
